crates/diesel_models/src/payment_attempt.rs

```diff
@@ -350,6 +350,15 @@ pub enum PaymentAttemptUpdate {
         authentication_id: Option<String>,
         updated_by: String,
     },
+    ManualUpdate {
+        status: Option<storage_enums::AttemptStatus>,
+        error_code: Option<Option<String>>,
+        error_message: Option<Option<String>>,
+        error_reason: Option<Option<String>>,
+        updated_by: String,
+        unified_code: Option<Option<String>>,
+        unified_message: Option<Option<String>>,
```

Comment:
why are these made Option<Option<>>, is there a way to unset these values from the api?